lib/v2/medium/router.js

```diff
@@ -0,0 +1,6 @@
+module.exports = (router) => {
+    router.get('/list/:user/:catalogId', require('./list'));
+    router.get('/for-you/:user', require('./for-you'));
+    router.get('/following/:user', require('./following'));
+    router.get('/tag/:user/:tag', require('./tag'));
```

Comment:
```suggestion
    router.get('/following/:user', require('./following'));
    router.get('/for-you/:user', require('./for-you'));
    router.get('/list/:user/:catalogId', require('./list'));
    router.get('/tag/:user/:tag', require('./tag'));
```
Please insert new entries in alphabetical order.